src/components/HomePage/HomePageHero.module.scss

```diff
@@ -0,0 +1,31 @@
+@use "src/styles/breakpoints";
+
+.outerContainer {
+  background-color: var(--color-background-alternative-faded);
+}
+
+.innerContainer {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  align-items: center;
+  padding-block-start: calc(2 * var(--spacing-mega));
+  padding-block-end: calc(1 * var(--spacing-mega));
+  padding-inline: var(--spacing-small);
+  @include breakpoints.tablet {
+    max-width: calc(25 * var(--spacing-mega));
+  }
+  margin-inline: auto;
+}
+
+.imageContainer {
+  height: calc(5 * var(--spacing-mega));
+  width: 100%;
+  max-width: calc(8 * var(--spacing-mega));
+  margin-block-end: var(--spacing-medium);
+  margin-block-start: var(--spacing-mega);
+}
+
+.quickLinksContainer {
+  margin-top: var(--spacing-mega);
```

Comment:
Can you run `yarn lint:scss` because it seems the linting is not working properly as this is a physical not logical value which should have been auto-fixed